Form/Type/OptionType.php

```diff
@@ -92,22 +100,11 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults(array(
             'data_class' => 'Hackzilla\Bundle\PasswordGeneratorBundle\Entity\Options',
             'csrf_protection' => false,
+            'generator' => null,
         ));
     }
 
-
-    /**
-     * {@inheritdoc}
-     */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
-    {
-        $this->configureOptions($resolver);
-    }
-
-    /**
-     * @return string
-     */
-    public function getName()
+    public function getBlockPrefix()
     {
         return '';
```

Comment:
If you support SF 2.7, you have to keep the `getName` method:

```php

    /**
     * {@inheritdoc}
     * 
     * SF < 2.8 BC
     */
    public function getName()
    {
        return $this->getBlockPrefix();
    }

    /**
     * {@inheritdoc}
     */
    public function getBlockPrefix()
    {
        return '';
    }
```

See: https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md#form